### PR TITLE
chore: removed enable_learners_tab_in_discussions_mfe flag dependency

### PR DESCRIPTION
**Pull request by ayesha-waris**
[INF-1185](https://2u-internal.atlassian.net/browse/INF-1185)

## Description

Removed `discussions.enable_learners_tab_in_discussions_mfe `dependency in MFE

### CODE REVIEW

File: src/discussions/discussions-home/DiscussionSidebar.jsx [hunk 1]
```diff
@@ -103,14 +102,12 @@ const DiscussionSidebar = ({ displaySidebar, postActionBarRef }) => {
           {ROUTES.TOPICS.PATH.map(path => (
             <Route key={path} path={path} element={<LegacyTopicsView />} />
           ))}
-          {redirectToLearnersTab && (
+          {(
             [ROUTES.LEARNERS.POSTS, ROUTES.LEARNERS.POSTS_EDIT].map((route) => (
               <Route key={route} path={route} element={<LearnerPostsView />} />
             ))
           )}
```
Comment (sundasnoreen12) [inline src/discussions/discussions-home/DiscussionSidebar.jsx:109]: remove square brackets.
       {
            [ROUTES.LEARNERS.POSTS, ROUTES.LEARNERS.POSTS_EDIT].map((route) => (
              <Route key={route} path={route} element={<LearnerPostsView />} />
            ))
          }

File: src/discussions/discussions-home/DiscussionSidebar.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ import Spinner from '../../components/Spinner';
 import { RequestStatus, Routes as ROUTES } from '../../data/constants';
 import { DiscussionContext } from '../common/context';
 import {
-  useContainerSize, useIsOnDesktop, useIsOnXLDesktop, useShowLearnersTab,
+  useContainerSize, useIsOnDesktop, useIsOnXLDesktop,
```
Comment (sundasnoreen12) [inline src/discussions/discussions-home/DiscussionSidebar.jsx:18]: move these into single line

**Review by sundasnoreen12 — CHANGES_REQUESTED** [review]
Please remove this flag dependency from test files also.
PostLink.test.jsx
PostsView.test.jsx
LearnersView.test.jsx
DiscussionsHome.test.jsx
AuthorLabel.test.jsx

**Review by awais-ansari — APPROVED** [review]
Just small nits as mentioned by Sundas other than LGTM.

**Review by sundasnoreen12 — APPROVED** [review]
PR LGTM. 
please rebase the PR with master

**Pull request merged**